config/prow/cluster/kubernetes_external_secrets.yaml

```diff
@@ -27,3 +26,16 @@ spec:
   - key: prometheus-alert-slack-post-testing-ops-secret-url
     name: url
     version: latest
+---
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: grafana
+  namespace: prow-monitoring
+spec:
+  backendType: gcpSecretsManager
+  projectId: k8s-prow
+  data:
+  - key: gke_k8s-prow_us-central1-f_prow__prow-monitoring__grafana
```

Comment:
This secret already exists in k8s-prow project, value was grabbed from the secret located in cluster